webExtensionApis/study/fakeApi.js

```diff
@@ -0,0 +1,248 @@
+/* eslint-disable */
+
```

Comment:
Note at top that this is a temporary file until the body of the `study` API methods are written in `./webExtensionApis/study/src/index.js`? Isn't this a generated file? Do we want to include a generated file in the repo?